provision-base.sh

```diff
@@ -128,13 +137,16 @@ if [ -f /vagrant/shared/tls/example-ca/example-ca-crt.pem ]; then
 fi
 
 # install iptables.
-apt-get install -y iptables
+apt-get install -y iptables systemd-resolved
+
+sed 's,^#DNS=.*,DNS=1.1.1.1 8.8.8.8,' -i /etc/systemd/resolved.conf
```

Comment:
why is this required? the VMs should use the DNS server from the host hypervisor.